lib/raygun.ts

```diff
@@ -133,7 +133,7 @@ class Raygun {
     return this;
   }
 
-  user(req?: Request): RawUserData | null {
+  user(req?: RequestParams): RawUserData | null {
```

Comment:
The original code mixed the `Request` type from Express with the internal `RequestParams` type, after upgrading dependencies the "compiler" didn't like that